invenio_app_rdm/theme/assets/semantic-ui/less/invenio_app_rdm/theme/collections/table.overrides

```diff
@@ -108,19 +108,24 @@
             }
 
             &[data-label]  {
-              display: inline-grid !important;
-              grid-template-columns: [col1-start] 30% [col2-start] 70% [end];
+              display: grid;
+              grid-template-columns: [col1-start] 20% [col2-start] 80% [end];
```

Comment:
30% had too much whitespace and 20% doesn't cause the label to go onto two lines on any admin panels